public/app/features/provisioning/JobStatus.tsx

```diff
@@ -43,11 +43,17 @@ export function JobStatus({ name, onStatusChange }: JobStatusProps) {
           </Text>
           <ProgressBar progress={job.status.progress} />
 
-          {job.status.state === 'success' && (
+          {job.status.state === 'success' ? (
             <Stack direction="column" gap={2}>
               {job.status.summary && <MigrationSummaryTable summary={job.status.summary} />}
               <RepositoryLink name={job.metadata?.labels?.repository} />
             </Stack>
+          ) : (
+            <>
+              <ControlledCollapse label="View details" isOpen={false}>
+                <pre>{JSON.stringify(job, null, ' ')}</pre>
+              </ControlledCollapse>
+            </>
```

Comment:
I think we still need to show the raw details until we are more sure that everything behaves as expected.  I have watched a few examples now where things are totally blank and require looking at the console/network tab to understand what may have happened